test/system/060-mount.bats

```diff
@@ -84,6 +84,24 @@ load helpers
     is "$output" "" "podman image mount, no args, after umount"
 }
 
+@test "podman run --mount ro=false " {
+      run_podman 1 run --rm -q --mount type=volume,destination=/path/in/container,readonly=true $IMAGE touch /path/in/container/a
+      is "$output" "touch: /path/in/container/a: Read-only file system"
+
+      run_podman 1 run --rm -q --mount type=volume,destination=/path/in/container,ro=true $IMAGE touch /path/in/container/a
+      is "$output" "touch: /path/in/container/a: Read-only file system"
+
+      run_podman 1 run --rm -q --mount type=volume,destination=/path/in/container,ro $IMAGE touch /path/in/container/a
+      is "$output" "touch: /path/in/container/a: Read-only file system"
+
+      run_podman 1 run --rm -q --mount type=volume,destination=/path/in/container,rw=false $IMAGE touch /path/in/container/a
+      is "$output" "touch: /path/in/container/a: Read-only file system"
+
+      run_podman run --rm -q --mount type=volume,destination=/path/in/container,rw=true $IMAGE touch /path/in/container/a
+      run_podman run --rm -q --mount type=volume,destination=/path/in/container,ro=false $IMAGE touch /path/in/container/a
+      run_podman run --rm -q --mount type=volume,destination=/path/in/container,readonly=false $IMAGE touch /path/in/container/a
+}
+
```

Comment:
Cleanup! My favorite, thank you! But as long as you're refactoring, why not make the tests a little more readable too?
```bash
@test "podman run --mount ro=false " {
    local volpath=/path/in/container
    local stdopts="type=volume,destination=$volpath"

    # Variations on a theme (not by Paganini). All of these should fail.
    for varopt in readonly=true ro=true ro rw=false;do
        run_podman 1 run --rm -q --mount $stdopts,$varopt $IMAGE touch $volpath/a
        is "$output" "touch: $volpath/a: Read-only file system" "with $varopt"
    done

    # All of these should pass
    for varopt in rw=true ro=false readonly=false;do
        run_podman run --rm -q --mount $stdopts,$varopt $IMAGE touch $volpath/a
    done
}
```

Reply:
Thanks I added a couple of more 
`readonly` to first test

`rw` to second test